features/dd-sdk-android-session-replay/build.gradle.kts

```diff
@@ -74,6 +74,12 @@ unMock {
     keep("android.widget.ImageView\$ScaleType")
     keep("android.graphics.Rect")
     keep("android.graphics.drawable.GradientDrawable")
+    keep("android.os.Handler")
+    keep("android.os.IMessenger")
+    keep("android.os.Looper")
+    keep("android.os.Message")
+    keep("android.os.MessageQueue")
+    keep("android.os.SystemProperties")
```

Comment:
this is not needed if `Looper` is not used

Reply:
Removed